### PR TITLE
Em 2683 update icp demo

### PR DESCRIPTION
**Pull request by meganaslattery**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EM-2683

### Change description ###

Dust down code for ICP demo

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/in-court/hearing-data.service.ts [hunk 1]
```diff
@@ -12,10 +12,14 @@ export class HearingDataService {
...
-        });
+        observer.next(resp.documents);
+        observer.complete();
+
+        // Old code used to call dm store again.
+        // this.docDataService.getDataFromUrls(resp.documents).subscribe(docs => {
+        //   observer.next(docs);
+        //   observer.complete();
+        // });
       });
```
Comment (tomxelliott) [inline src/app/in-court/hearing-data.service.ts]: Can we get rid of the commented out code or could it come in handy at a later date?

**Review by tomxelliott — APPROVED** [review]

File: src/app/in-court/hearing-data.service.ts [hunk 2]
```diff
@@ -12,10 +12,14 @@ export class HearingDataService {
   public loadHearingDetails(sessionId: string): Observable<any[]> {
     return new Observable<any[]>(observer => {
       this.http.get<any>(`/icp/sessions/${sessionId}`).subscribe(resp => {
-        this.docDataService.getDataFromUrls(resp.documents).subscribe(docs => {
-          observer.next(docs);
-          observer.complete();
-        });
+        observer.next(resp.documents);
+        observer.complete();
```
Comment (benouaer) [inline src/app/in-court/hearing-data.service.ts]: not sure what this is doing and whether it's needed:
observer.next(resp.documents);
observer.complete();

File: src/app/in-court/hearing-data.service.ts (hunk 1 above)
Comment (meganaslattery) [reply src/app/in-court/hearing-data.service.ts]: I wasn't sure if it was useful so I kept it... probably can get rid of it though :)

**Review by benouaer — APPROVED** [review]

File: src/app/in-court/hearing-data.service.ts (hunk 2 above)
Comment (meganaslattery) [reply src/app/in-court/hearing-data.service.ts]: Probably not needed anymore, I just copied part of the old code. I'll change that now

**Pull request closed without merging**